crates/db_schema/src/impls/actor_language.rs

```diff
@@ -24,6 +24,7 @@ impl LocalUserLanguage {
         Box::pin(async move {
           let langs = local_user_language
             .filter(local_user_id.eq(for_local_user_id))
+            .order(id)
```

Comment:
Should also order by `local_user_language::language_id`

The `local_user_language` is a join table, so its id is meaningless.